public/app/features/org/SelectOrgPage.tsx

```diff
@@ -18,29 +18,33 @@ const navModel = {
   },
 };
 
-const getUserOrgs = async () => {
-  return await getBackendSrv().get('/api/user/orgs');
-};
+function mapStateToProps(state: StoreState) {
```

Comment:
here we define `function mapStateToProps`, and below we define `const mapDispatchToProps`. doesn't really make a difference, but we should be consistent